topics/ecology/tutorials/lotus2-identifying-fungi/tutorial.md

```diff
@@ -0,0 +1,462 @@
+> > ```
+> >
+> {: .solution}
+>
+{: .question}
+
+
+# Run LotuS2 on your own data
+
+To run the LotuS2 workflow on your own fungal data, you only need the sequencing fastq files and some information about what primers were used.
+
+In the example dataset, we used the ITS3/ITS4 primer pair to sequence the ITS2 reqion, so our Forward primer was `GCATCGATGAAGAACGCAGC` and the Reverse primer was `TCCTCCGCTTATTGATATGC`
+
+When you want to process your own sequencing files, you can specify the Forward and Reverse primer in the Galaxy LotuS2 Parameters directly in the options if all your samples have the same forward and reverse primers:
+  - *"Forward primer used to amplify DNA region (optional) - optional"*: `GCATCGATGAAGAACGCAGC`
+  - *"Reverse primer used to amplify DNA region (optional) - optional"*: `TCCTCCGCTTATTGATATGC`
+
+We advise adding metadata to the phyloseq object after running LotuS2, so that a large mapping file with all metadata does not have to be created. See future tutorials on adding metadata from a .csv file to a phyloseq object in R.
+
+## Create your own mapping tsv file
+
+Mapping files for LotuS2 have to be in tab-separated-values format. You can create this file in any spreadsheet sofware, eg: Microsoft Excel or Google Sheets. The important thing to note is that when you save or export the file, you should select the TSV format (called "Tab separated values" in Google Sheets or "Tab-delimited text .txt" in Excel)
+
+LotuS2 allows many columns in the mapping tsv file according to the [specification](https://lotus2.earlham.ac.uk//main.php?site=documentation#mappingfile). However, for simplicity, we recommend this format for sequencing data using the SPUN protocol:
+
+- Only 1 header row that begins with `#SampleID`
+- One row per sample
+- The only essential columns are `#SampleID`, `fastqFile`
+- If you have the same ForwardPrimer and ReversePrimer in all your samples, then you can provide them in the LotuS2 Galaxy tool and delete the ForwardPrimer and ReversePrimer columns
+- If you have any additional metadata variables that you want to analyse downstream using R, then you can provide them as columns
+- We recommend using "SequencingRun" and labelling with the name of the sequencing run, which is arbitrary. Something like "Scripps run 1". This is because if samples were run across multiple different sequencing runs it is important for LotuS2 to factor this into analyses. If samples were all run on the same run, write the same for every sample.
+
+For more information, see the [LotuS2 Mapping File Format Documentation](https://lotus2.earlham.ac.uk/main.php?site=documentation#mappingfile)
+
+In the exercise below you will create your own mapping tsv file for a new run.
+
+> <hands-on-title>Create and use your own mapping tsv file</hands-on-title>
+>
+> Steps:
+>
+> 1. Import history from: [mapping example history](https://usegalaxy.eu/u/sujai_spun_earth/h/identifying-mf-from-its2-sequencing-using-lotus2--create-mapping)
```

Comment:
again this will only be useful for learners using Galaxy EU. Could you add the files to Zenodo and provide the relevant import instructions?

Reply:
+1

Reply:
@shiltemann I have created a new zenodo record at https://zenodo.org/records/13848713 but given that this tutorial only runs on usegalaxy.eu for now, I'd like to keep this import history step as a quick way for new learners to get started.

Once LotuS2 works on other usegalaxy servers, I will update this tutorial to follow your suggestion of keeping it server agnostic! :-)